Cargo.toml

```diff
@@ -17,44 +17,39 @@ homepage = "https://github.com/matter-labs/teepot"
 
 [workspace.dependencies]
 actix-http = "3"
-actix-tls = "3"
 actix-web = { version = "4.5", features = ["rustls-0_22"] }
 anyhow = "1.0.82"
 awc = { version = "3.4", features = ["rustls-0_22-webpki-roots"] }
 base64 = "0.22.0"
-bitflags = "2.5"
 bytemuck = { version = "1.15.0", features = ["derive", "min_const_generics", "extern_crate_std"] }
 bytes = "1"
 clap = { version = "4.5", features = ["std", "derive", "env", "error-context", "help", "usage", "wrap_help"], default-features = false }
 const-oid = { version = "0.9", default-features = false }
 ctrlc = "3.4"
-der = "0.7.9"
+dcap-qvl = "0.1.6"
```

Comment:
It’s a very small and immature repo developed by Phala Network: https://github.com/Phala-Network/dcap-qvl. Is there an alternative? (TDX is still in its early stages, so I guess there might not be.)

Reply:
well, we could steal the quote parsing stuff... and maintain it ourselves...